src/Umbraco.Cms.Integrations.Automation.Zapier/Services/ZapierFormService.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+#if NETFRAMEWORK
+using System.Web.Mvc;
+#endif
+
+using Umbraco.Cms.Integrations.Automation.Zapier.Helpers;
+using Umbraco.Cms.Integrations.Automation.Zapier.Models.Dtos;
+
+namespace Umbraco.Cms.Integrations.Automation.Zapier.Services
+{
+    public class ZapierFormService
```

Comment:
You could move this (and the `FormController`) to the forms integration, and then can avoid using reflection.  Up to you now you've done it, but given we need it for the event handling we could simplify retrieving the forms as well.

The only thing you'd need to keep here in this integration is `FormsHelper.IsFormsExtensionInstalled()`.

Reply:
Updated.